cpp/src/arrow/compute/api_vector.h

```diff
@@ -131,8 +165,6 @@ class ARROW_EXPORT PartitionNthOptions : public FunctionOptions {
   int64_t pivot;
 };
 
-/// @}
```

Comment:
Did you mean to remove this? This tells Doxygen where a group ends.

Reply:
ohhh thanks I
 thought it was an error